webui/react/src/pages/F_ExpList/F_ExperimentList.tsx

```diff
@@ -724,7 +724,7 @@ const F_ExperimentList: React.FC<Props> = ({ project }) => {
   );
 
   const experimentsIfLoaded = useMemo(
-    () => (isLoading ? [NotLoaded] : experiments),
+    () => (isLoading ? INITIAL_LOADING_EXPERIMENTS : experiments),
```

Comment:
if `isLoading`is true, isn't `experiments` already equal to `INITIAL_LOADING_EXPERIMENTS`?

Reply:
That's right, but I think we specifically changed this in this [PR](https://github.com/determined-ai/determined/pull/7076)

Reply:
right, but the reason we memoized the value in that pr is because the value when loading was `[NotLoaded]`, so we didn't want to recreate the array every render. if we just pass `experiments` through now, it should have the same effect, no?

Reply:
Hi Nick @mapmeld, I wonder if you still remember what was the issue with just passing the `experiments`, I do not  seem to run into anything, has this issue been solved somewhere else?

Reply:
I don't remember. Reading the code now, the change doesn't make a lot of sense to me